Carthage.xcodeproj/xcshareddata/xcschemes/carthage.xcscheme

```diff
@@ -65,7 +65,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      launchStyle = "0"
```

Comment:
is this intentional?

Reply:
No this is not correct, will revert that change